src/Test.elm

```diff
@@ -99,6 +100,34 @@ test desc thunk =
     Internal.Labeled desc (Internal.Test (\_ _ -> [ thunk () ]))
 
 
+{-| Returns a [`Test`](#Test) that is "TODO" (not yet implemented). These tests
+always fail, but test runners will only include them in their output if there
+are no other failures.
+
+The intended use is when you're brainstorming lots of tests you'd like to write,
+but you can't implement them all at once. When you replace `todo` with a real
+test, you'll be able to see if it fails without clutter from tests still not
+implemented. But, unlike leaving yourself comments, you'll be prompted to
+implement these tests because your suite will fail.
+
+    describe "a new thing"
+        [ todo "does what is expected in the common case"
+        , todo "correctly handles an edge case I just thought of"
+        ]
+-}
```

Comment:
I think it would be a good idea to mention two things:

- `todo` tests are usually not meant to be committed to a code-base. It's only a tool for development.
- `todo` tests are sometimes called `pending` in other test-frameworks. (This comment would also be helpful for people searching for the word `pending` in the docs or on google.)